upgrade_kafka_broker.yml

```diff
@@ -41,13 +48,19 @@
         line: "inter.broker.protocol.version={{inter_broker_protocol_version}}"
         regexp: inter.broker.protocol.version.*
         state: present
+      # notify:
+      #   - Restart Kafka
```

Comment:
@JumaX Curious your take on these kafka properties/ if a restart is necessary. The scenario is this: kafka is at version 2.3, and log.message.format.version is not set. The playbook will set it to 2.3, but wont restart kafka (bc its not necessary). Then later kafka gets upgraded to 2.4 and on that restart its important the version stays at 2.3.

Reply:
@domenicbove so if I understand, we are simply writing the property so we have it for the next upgrade, as the Broker is already set to that properties value.  In this case, not restarting makes sense.